scripts/zones/Bastok_Markets/npcs/Nbu_Latteh.lua

```diff
@@ -55,7 +55,7 @@ function onEventFinish(player,csid,option)
             player:setVar("MomTheAdventurer_Event",1);
             player:addItem(4096);
             player:messageSpecial(ITEM_OBTAINED,4096); -- Fire Crystal
-            if (player:questStatus(BASTOK,MOM_THE_ADVENTURER == QUEST_AVAILABLE)) then
```

Comment:
I'm so sorry.

![image](https://user-images.githubusercontent.com/3845823/36286899-1a5dbc02-12f5-11e8-8b6c-4725f3ef16d5.png)

Reply:
Eee!.
I didn't mean to make the contributor seppuku I swears!